flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/dataset/DataSetSingleRowJoin.scala

```diff
@@ -44,9 +45,11 @@ class DataSetSingleRowJoin(
     leftNode: RelNode,
     rightNode: RelNode,
     leftIsSingle: Boolean,
+    rightIsSingle: Boolean,
     rowRelDataType: RelDataType,
     joinCondition: RexNode,
     joinRowType: RelDataType,
+    joinType: JoinRelType,
```

Comment:
@fhueske we already have `.item("joinType", joinTypeToString)` in `explainTerms`. Did you mean change `joinTypeToString` ("NestedLoopJoin") to `joinType.toString`?